### ISSUE TITLE
TF 2.0 How to set a different learning rate for each layer?

### ISSUE THREAD

**Issue by yw155** [tensorflow/community#161]
Hi, I would like to ask a question about how to set different learning rates for several layers. In my implementation, I would like to set one learning rate for some layers while the other learning rate for some other layers. Is there some example codes? Thanks in advance.

**Comment by ewilderj** [comment]
Sorry, this repo is not monitored for questions. Please ask on Stack Overflow.
